Artifacts/windows-slack/Artifactfile.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2015-01-01/dtlArtifacts.json",
+  "title": "Slack",
+  "description": "Installs Slack using the Chocolatey package manager",
+  "tags": [
+    "Windows",
+    "Slack"
+  ],
+  "iconUri": "https://chocolatey.org/content/packageimages/slack.1.0.0.png",
+  "targetOsType": "Windows",
+   "parameters": {
+        "installUsername": {
```

Comment:
I see that this artifact requires the user to pass in a username and password (for installing slack under a specific user's credentials). Would it be possible to eliminate the need for these parameters?

The general pattern that most artifacts follow is creation of a temporary admin account to install the chocolatey package (Example: https://github.com/Azure/azure-devtestlab/tree/master/Artifacts/windows-7zip). This way, the user is not required to pass in a user name and password.

Will that work for this artifact?